apps/web/app/app.dub.co/(dashboard)/[slug]/(ee)/program/bounties/bounty-list.tsx

```
@@ -0,0 +1,71 @@
+                <BountyCard key={bounty.id} bounty={bounty} />
+              ))}
+        </div>
+      ) : (
+        <AnimatedEmptyState
+          title="No bounties active"
+          description={
+            <>
+              This program doesn't have any active bounties.{" "}
+              <a
+                href="https://dub.co/help/article/bounties"
+                target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)